sklearn/metrics/cluster/_supervised.py

```diff
@@ -47,10 +49,19 @@ def check_clusterings(labels_true, labels_pred):
     labels_true = check_array(
         labels_true, ensure_2d=False, ensure_min_samples=0, dtype=None,
     )
+
     labels_pred = check_array(
         labels_pred, ensure_2d=False, ensure_min_samples=0, dtype=None,
     )
 
+    type_label = type_of_target(labels_true)
+    type_pred = type_of_target(labels_pred)
+
+    if 'continuous' in (type_pred, type_label):
+        msg = f'Expects discrete values but received {type_label} ' \
```

Comment:
Can we put a subject on this, e.g. "Clustering metrics expect ..."? It can be hard to determine the context of a warning.

Reply:
@jnothman done